cadquery/cq.py

```diff
@@ -2239,22 +2241,32 @@ def rect(
         Creates 4 circles at the corners of a square centered on the origin.
 
         Future Enhancements:
-            better way to handle forConstruction
-            project points not in the workplane plane onto the workplane plane
+            * better way to handle forConstruction
```

Comment:
I think this is a hold-over from long ago, and I'm not even sure what it means. Does it make sense to keep it?

Reply:
Yeah, I think it should be removed. AFAIK there are no issues open for `forConstruction` and no plans to change it's action.